src/ci_cd.tex

```diff
@@ -91,16 +91,16 @@ \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Tes
 %%%%%%%% RIGHT COLUMN
 
 \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Build library}
-Leveraging pre-built components and artifacts to expedite software development,
+Leverages pre-built components and artifacts to expedite software development,
 enhancing efficiency and scalability in the creation of complex applications.
 
 \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Release}
-Leveraging pre-built components and artifacts to expedite software development,
+Leverages pre-built components and artifacts to expedite software development,
 enhancing efficiency and scalability in the creation of complex applications.
 
```

Comment:
I think the main issue is that the Release section has just been copy-pasted. Could you adapt it @jorgepiloto?

Reply:
Let's do it in a follow up PR